lib/board.rb

```diff
@@ -27,4 +30,22 @@ def make_free_positions_hash(blanks)
     end
     hash
   end
+
+  def board_state(symbol)
+    return nil if num_of_played_positions < board.size * 2 - 1
```

Comment:
`num_of_played_positions < board.size * 2 - 1` might be a good place to introduce an explaining variable.

Reply:
What would happen if the game is over before all the positions are taken? For example,

```
o o +
x x x
+ + +
```

Reply:
Resolved at 4366f41b26aec5c9cbf1f087275c1ec47033cacb